scripts/copy-dist.js

```diff
@@ -77,7 +77,7 @@ await copy({
     // seed assets - kai binaries
     {
       context: "downloaded_assets/kai",
-      src: ["**/*", "!**/*.zip"],
+      src: ["kai-rpc-server*/kai*", "!**/*.zip"],
```

Comment:
Don't 100% understand this change. Does this exclude the analyzer binary from being copied?

Reply:
Ah! The zip file for the kai workflow artifact includes a bunch more stuff now.  This glob change will have it only pickup the binaries.